### ISSUE TITLE
What are parameters in solver.prototxt?

### ISSUE THREAD

**Issue by mjohn123** [shicai/DenseNet-Caffe#3]
Thanks for sharing the work.  Do you use same parameter settings in the https://github.com/liuzhuang13/DenseNetCaffe for optimization? I ask it because the learning rate of the reference one is so high (about 0.1). In additions, do you get the performance using your prototxt it for CIFAR dataset? Thanks

**Comment by shicai** [comment]
The learning rate is ok for me.
It is quite easy to reproduce the results from the paper.

**Comment by mjohn123** [comment]
Thanks. I think the learning rate depends on dataset and solver optimization method. 
In all prototxts, the dropout did not use. Did you try to add the dropout as 0.2 in BC model and get the result? How much it improve?

**Comment by shicai** [comment]
I have trained DenseNet on CIFAR10 and ImageNet, and lr=0.1 works for me.
I did not try to use dropout for DenseNet.
You'd better ask the author. @liuzhuang13
